compiler/vm/vmconv.nim

```diff
@@ -1,5 +1,14 @@
 import ast/ast
 
+import vm/[
+  vmdef,
+  vmobjects
+]
+
+# TODO: correctly and fully supporting every type with `fromLit`/`toLit` seems
```

Comment:
`toLit`/`fromLit` are not needed nor used by the VM but are a marshalling/unmarshalling convenience utility used by `VmCallback`'s:
```nim
let atom = toLit(10)
# Instead of:
# let value = VmObjectAtom(kind: akInt, intVal: 10)

let i = fromLit(atom)
# Instead of:
# let i = atom.i
```
While both examples are more or less trivial for ints, you'd have to write more manual code for complex objects.

Before this PR they also only supported a few select cases with an "add as needed" comment